wafer/pages/models.py

```diff
@@ -68,6 +78,24 @@ def get_absolute_url(self):
         url = "/".join(self.get_path())
         return reverse('wafer_page', args=(url,))
 
+    def cached_render(self):
+        if self.cache_time < 0:
+            return self.content.rendered
+        cache = caches[self.cache_name]
+        cache_key = self.get_absolute_url()
```

Comment:
If multiple parts of wafer are using the cache for different things, should we be prefixing the key names with some sort of namespace string?